license_sh/__main__.py

```diff
@@ -14,6 +14,7 @@
   -t --tree                           Show full dependency tree.
   -d --debug                          Debug mode
   -p --project <project_type>         Run only specific project [yarn | npm | maven | pipenv]
+  -a --analyze                        Analyze dependencies. Suported: npm, yarn
```

Comment:
I've been out for a while, so I'd appreciate if I could get a more detailed explanation of this flag. Maybe in a docs? Or better inline explanation of what `--analyze` is